src/Ml.Cli.WebApp/Server/Datasets/Cmd/CreateDatasetCmd.cs

```diff
@@ -23,6 +23,8 @@ public record CreateDatasetCmdInput
     public string GroupId { get; set; }
 
     [MaxLength(32)] [MinLength(1)] public string CreatorNameIdentifier { get; set; }
+    
+    public string ImportedDatasetName { get; set; }
```

Comment:
You must secure the input, for example :

    [MaxLength(48)]
    [MinLength(3)]
    [RegularExpression(@"^[a-zA-Z0-9-_]*$")]